libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/NavigatorLoader.kt

```diff
@@ -23,10 +24,15 @@ internal object NavigatorLoader {
         System.loadLibrary("navigator-android")
     }
 
-    fun createNavigator(deviceProfile: DeviceProfile, logger: Logger?): Navigator {
+    fun createNavigator(
+        deviceProfile: DeviceProfile,
+        navigatorConfig: NavigatorConfig,
```

Comment:
Is there a need/plan for the platform SDKs to expose the navigator config fields in a readable or serialized form @mskurydin?